### ISSUE TITLE
Create an F1 Score Metric for Binary Classification

### ISSUE THREAD

**Issue by mfrasco** [mfrasco/Metrics#19]
Currently, the function `f1` is used for information retrieval problems. However, I really want a function called `f1_score` that can be used for binary classification problems. This function should take two arguments, `predicted` and `actual` and compute the f1 score by calculating the point-wise precision and recall of the predictions.

**Comment by adam-m-mcelhinney** [comment]
@mfrasco, I would propose an fbeta_score function that accepts an optional beta arugment with a default set to one. 

http://scikit-learn.org/stable/modules/generated/sklearn.metrics.fbeta_score.html
https://en.wikipedia.org/wiki/F1_score
